clients/client/src/main/java/org/eclipse/sw360/clients/adapter/SW360ConnectionFactory.java

```diff
@@ -69,6 +70,12 @@ public SW360Connection newConnection(SW360ClientConfig config) {
                 SyncClientAdapterHandler.newHandler(SW360ProjectClientAdapter.class,
                         SW360ProjectClientAdapterAsync.class, projectAdapterAsync);
 
+        SW360VulnerabilityClient vulnerabilityClient = new SW360VulnerabilityClient(config, tokenProvider);
+        SW360VulnerabilityClientAdapterAsync vulnerabilityAdapterAsync = new SW360VulnerabilityClientAdapterAsyncImpl(vulnerabilityClient);
+        SW360VulnerabilityClientAdapter vulnerabilityAdapterSync =
```

Comment:
Can you please check the vulnerabilityAdapterSync, looks like return statement is missing for this.

Reply:
added method to return sync adapter